utils/eval_utils.py

```diff
@@ -169,8 +169,8 @@ def save_predictions(self, start_datetime=None, run_time=0,
             json_dict['machine_labels'].append(params)
 
         # Save as json file
-        
-        json_fname = os.path.join(dest_dir, 'predictions.json')
+        unique_id = datetime.now().strftime('%Y%m%d%H%M%S')
+        json_fname = os.path.join(dest_dir, 'predictions_'+unique_id+'.json')
```

Comment:
Rather than using a unique-id, let's use the original csv filename and set it up as
<CSV basename>-predictions.json

Reply:
I agree to add original csv filename. But I think it would be better to also add model id into it, since we might deploy different models on same csv file. Maybe something like filename-model_id-predictions.json.